src/CMakeLists.txt

```diff
@@ -0,0 +1,323 @@
+cmake_minimum_required(VERSION 3.2)
+project(QZXing)
+
+find_package(Qt5 REQUIRED Core)
+find_package(Qt5 REQUIRED Network)
+find_package(Qt5 REQUIRED Xml)
+find_package(Qt5 REQUIRED Multimedia)
+find_package(Qt5 REQUIRED MultimediaWidgets Widgets)
+find_package(Qt5 REQUIRED Svg Quick QuickControls2)
+
+set(CMAKE_AUTOMOC ON)
+#set(CMAKE_BUILD_TYPE Debug)
+add_subdirectory(zxing)
```

Comment:
you did not include a zxing/CMakeLists.txt, did you miss it?

Reply:
It was my mistake. zxing/CMakeLists.txt Was not necessary.